tools/doc/apilinks.js

```diff
@@ -54,10 +54,15 @@ inputs.forEach((file) => {
 
   // Parse source.
   const source = fs.readFileSync(file, 'utf8');
-  const ast = acorn.parse(
-    source,
-    { allowReturnOutsideFunction: true, ecmaVersion: 10, locations: true });
-  const program = ast.body;
+  let program;
+  try {
+    const ast = acorn.parse(
+      source,
+      { allowReturnOutsideFunction: true, ecmaVersion: 10, locations: true });
+    program = ast.body;
+  } catch {
+    return;
+  }
```

Comment:
For whatever bug this change fixes, is it possible to add a test case for it to `test/doctool/test-apilinks.js`?

Reply:
It can't parse private fields. I suppose I can add a test.